rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -353,149 +338,10 @@ upgrade_ncf() {
 
 # Upgrade the file logback.xml
 upgrade_logback_xml() {
-  # - 2.8.0 : Update logback.xml in order to have information about change requests
-  STEP="Update logback.xml in order to have information about change requests"
-
-  if ! perl -p0e 's/\n//g' < /opt/rudder/etc/logback.xml | perl -p0e 's/<!--.(?:(?!-->).)*-->//g' | perl -p0e 's/> *</></g' | grep -E '<logger name="changeRequest" level="[^"]+" additivity="false">' >/dev/null 2>&1
-  then
-    sed -i 's%^ *</configuration>%   <logger name="changeRequest" level="info" additivity="false">\n     <appender-ref ref="OPSLOG" />\n     <appender-ref ref="STDOUT" />\n   </logger>\n</configuration>%' /opt/rudder/etc/logback.xml
-  fi
 }
```

Comment:
I prefer keeping the function so we remember there can me some steps of migrating logback.xml